packages/edgeworth/src/synthesis/Synthesizer.ts

```diff
@@ -432,8 +433,8 @@ export class Synthesizer {
    * @param numProgs number of Substance programs to generate
    * @returns an array of Substance programs and some metadata (e.g. mutation operation record)
    */
-  generateSubstances = (numProgs: number): SynthesizedSubstance[] =>
-    _.times(numProgs, (n: number) => {
+  generateSubstances = (numProgs: number): SynthesizedSubstance[] => {
+    const iteratee = (n: number) => {
```

Comment:
```suggestion
    const oneSubstance = () => {
```

Slightly more readable name